seq2science/schemas/config/gene_expression.schema.yaml

```diff
@@ -45,16 +45,20 @@ properties:
         htseq_flags: ''
 
   tximeta:
-    description: tximeta is used to convert salmon output (estimated transcript abundances) to a gene counts matrix. Currently, these values can be ignored.
+    description: tximeta is used to convert salmon output to gene counts. If you are using an assembly from Ensembl or GENCODE, you can refine the output by specifying these fields.
     properties:
       source:
-        default: Ensembl # the source of transcriptome (only works with "Ensembl")
+        description: the source of transcriptome
+        enum: [Ensembl, GENCODE, de-novo]
+        default: de-novo
       organism:
-        default: tximeta_file # organism (e.g. "Homo sapiens")
+        description: organism (e.g. "Homo sapiens")
+        default: tximeta_file
       release:
-        default: 42 # release number (e.g. "27")
+        description: release number of the source database (e.g. "101")
+        default: 42
     default:
-      source: Ensembl
+      source: de-novo
```

Comment:
@siebrenf same here?

Reply:
See below. The fix was to use the new "de-novo" option in this version of tximeta. lets see if it still works!